web/apps/labelstudio/src/pages/CreateProject/Config/Preview.jsx

```diff
@@ -59,7 +59,7 @@ export const Preview = ({ config, data, error, loading, project }) => {
         const lsf = new window.LabelStudio(rootRef.current, {
           config,
           task,
-          interfaces: ["side-column", "annotations:comments", "comments:resolve-any"],
+          interfaces: ["side-column", "comments:resolve-any"],
```

Comment:
then it would be reasonable to also remove `comments:resolve-any`
```suggestion
          interfaces: ["side-column"],
```

Reply:
Good catch, there would be no reason to have resolve:any here probably ever since its the preview and you can't comment there.